style.css

```diff
@@ -260,3 +260,84 @@ a:hover {
   width: 40px;
   margin-right: 20px;
 }
+
+/* contact section */
+.contact-form {
+  background-image: url("./images/contact\ form\ shapes-mobile.png");
+  background-repeat: no-repeat;
+  background-position: right;
+  background-color: #6070ff;
+  height: 779px;
+}
+
+.contact-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 80px;
+  color: white;
+  gap: 12px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.contact-text h2 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  height: 56px;
+}
+
+.contact-text p {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  display: flex;
+  text-align: center;
+}
+
+.forms {
+  display: flex;
+  flex-direction: column;
+}
+
+.forms input {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin: 20px 24px;
+  padding: 25px 70px;
+  text-align: left;
+}
+
+.forms textarea {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin: 20px 24px;
+  padding: 15px 80px;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 16px;
+  width: 140px;
+  height: 48px;
+  margin: 0 24px;
+  border-radius: 8px;
+  border: 1px solid #ffff;
+}
+
+.btn:hover {
+  background-color: rgb(118, 190, 118);
+  border: 1px solid rgb(118, 190, 118);
```

Comment:
- [ Optional]  You did a good job however I suggest that you change your button colors to match the Figma design.